frontend.bat

```diff
@@ -523,13 +524,17 @@ for /F "tokens=1,2 delims=." %%a in ("!vsver!") do (
     rem https://github.com/3F/hMSBuild/issues/3
     set vsver=%%~a.0
 )
-set _msbp=!vspath!\MSBuild\!vsver!\Bin
+REM VS2019 changes the path
+if !vsver! geq 16 set vsver=Current
+
+if not exist "!vspath!\MSBuild\!vsver!\Bin" set "%3=" & exit /B 0
```

Comment:
I think we need to return the non-zero status code to avoid future misunderstanding when using :vswmsb. Although, this `call` will check definition of {out:msbfile} argument, but...

above, I'm watching same reason:

```bat
set "%3=" & exit /B %ERROR_PATH_NOT_FOUND%
```

Reply:
updated
Path was empty before 16.0->Current change, should be an error